test/src/main/java/org/apache/accumulo/test/functional/FunctionalTestUtils.java

```diff
@@ -218,4 +219,30 @@ private static FateStatus getFateStatus(ClientContext context, AccumuloCluster c
       throw new RuntimeException(e);
     }
   }
+
+  static long checkFlushId(AccumuloClient c, String tableName, long prevFlushID) throws Exception {
+    try (Scanner scanner = c.createScanner(MetadataTable.NAME, Authorizations.EMPTY)) {
+      String tableId = c.tableOperations().tableIdMap().get(tableName);
+      scanner.setRange(new Range(new Text(tableId + ";"), true, new Text(tableId + "<"), true));
+      scanner.fetchColumnFamily(TabletsSection.ServerColumnFamily.NAME);
+      TabletsSection.ServerColumnFamily.FLUSH_COLUMN.fetch(scanner);
+
+      long flushId = -1L;
+      for (Entry<Key,Value> entry : scanner) {
+        Key key = entry.getKey();
+        String val = entry.getValue().toString();
+
+        if (key.getColumnQualifierData().toString().equals(FLUSH_QUAL)) {
+          flushId = Long.parseLong(val);
```

Comment:
Does it matter if diff tablets have diff flush ids?

Reply:
Are you asking if we should check if the flush ids are different for all the tablets?

Reply:
Yeah that is a good idea.